src/main/java/citadel/Citadel.java

```diff
@@ -0,0 +1,146 @@
+package citadel;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+import citadel.Task.TaskList;
+import citadel.commands.Commands;
+import citadel.commands.DeleteTask;
+import citadel.commands.FindTask;
+import citadel.commands.HandleDeadline;
+import citadel.commands.HandleEvent;
+import citadel.commands.HandleTodo;
+import citadel.commands.MarkTask;
+import citadel.commands.UnmarkTask;
+import citadel.exception.CitadelException;
+import citadel.exception.CitadelInvalidCommandException;
+import citadel.ui.TextUI;
+
+/**
+ * The {@code Citadel} class represents the main entry point of the Citadel application.
+ * It manages user interactions, processes commands, and coordinates tasks and data storage.
+ */
+public class Citadel {
+
+    /** The list of tasks managed by the application. */
+    private static TaskList items = new TaskList();
+
+    /** The storage system used to persist tasks between sessions. */
+    private static final Storage db = new Storage("data/citadel");
+
+    /** The user interface used to interact with the user. */
```

Comment:
Could this be replaced with normal comments instead of Javadoc comments?

Reply:
Good catch